assets/section-footer.css

```diff
@@ -463,16 +464,54 @@ noscript .localization-selector.link {
   }
 }
 
-.footer-block-image > img {
+.brand-information__image-wrapper {
+  box-shadow: var(--media-shadow-horizontal-offset)
+    var(--media-shadow-vertical-offset)
+    var(--media-shadow-blur-radius)
+    rgba(var(--color-shadow), var(--media-shadow-opacity));
+  margin-bottom: calc(2rem + var(--media-shadow-vertical-offset) * var(--media-shadow-visible));
```

Comment:
this doesn't work well when the vertical offset is a negative value. 
I think on the product page we did it using `padding` so that when it's a negative value, nothing happens 🤔 
But we could also use a `max()`. `max((var(--media-shadow-vertical-offset) * var(--media-shadow-visible)), 0rem)`